docs/application/native/guides/multimedia/media-content.md

```diff
@@ -361,7 +368,9 @@ To find albums in the system and filter the results:
     media_album_foreach_album_from_db(filter, media_album_list_cb, NULL);
     ```
 
-	You can also use the `media_album_get_album_count_from_db()` function to get the number of albums available.The `media_album_foreach_album_from_db()` function requires a callback, which is called for every found album. The function is synchronous, and it blocks until the callback has been called for all albums or the callback returns `false`. For information on how to implement the callback, see [Reading Album Information](#read_album).
+	You can also use the `media_album_get_album_count_from_db()` function to get the number of albums available.
+
+    The `media_album_foreach_album_from_db()` function requires a callback, which is called for every found album. The function is synchronous, and it blocks until the callback has been called for all albums or the callback returns `false`. For information on how to implement the callback, see [Reading Album Information](#read_album).
 
 3. Destroy the filter:
 
```

Comment:
line 396: The separated lines are merged as one line.
line 399: Add 2 spaces at the beginning.(media_content_error_e ret)
line 436: This is not included in Note.(from the line 436 to 438)
line 441: Add 1 space at the beginning.(/* Get media count in)
line 633: Add 1 space at the beginning.(return true;)
line 727: Move to the previous line.
line 739: Move to the previous line.
line 745: The separated lines are merged as one line.
line 795: Move to the previous line.
line 897: Add 1 space at the beginning.(from the line 897 to 905)
line 903: Move to the previous line.
line 1023: Move to the previous line.
line 1056: Move to the previous line.
line 1087: Move to the previous line.
line 1371: Move to the previous line.
line 1525: Move to the previous line.
line 1528: Move to the previous line.
line 1533: Move to the previous line.
line 1552: Move to the previous line.
line 1554: Move to the previous line.
line 1556: Move to the previous line.
line 1558: Move to the previous line.
line 1653: Move to the previous line.
line 1807: The separated lines are merged as one line.
line 1836: The separated lines are merged as one line.
line 1839: The separated lines are merged as one line.
line 1847: The separated lines are merged as one line.
line 1858: The separated lines are merged as one line.
line 1866: The separated lines are merged as one line.
line 1867: Remove this line.
line 1868: Remove this line.
line 1876: The separated lines are merged as one line.